.github/scripts/check-updater.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+#
+set -e
+# shellcheck source=.github/scripts/functions.sh
+. "$(dirname "$0")/functions.sh"
+
+NAME="${NAME:-netdata}"
+VERSION="${VERSION:-"$(git describe --always)"}"
+BASENAME="$NAME-$VERSION"
```

Comment:
Shellcheck is correct that this appears to be unused. It should either be removed, or if used externally exported.